checker/src/main/java/org/checkerframework/checker/calledmethods/CalledMethodsTransfer.java

```diff
@@ -0,0 +1,40 @@
+package org.checkerframework.checker.calledmethods;
+
+import org.checkerframework.common.accumulation.AccumulationTransfer;
+import org.checkerframework.dataflow.analysis.TransferInput;
+import org.checkerframework.dataflow.analysis.TransferResult;
+import org.checkerframework.dataflow.cfg.node.MethodInvocationNode;
+import org.checkerframework.dataflow.cfg.node.Node;
+import org.checkerframework.framework.flow.CFAnalysis;
+import org.checkerframework.framework.flow.CFStore;
+import org.checkerframework.framework.flow.CFValue;
+
+/** A basic transfer function that accumulates the names of methods called. */
+public class CalledMethodsTransfer extends AccumulationTransfer {
+
+    /** The type factory. */
+    private final CalledMethodsAnnotatedTypeFactory atypeFactory;
+
+    /**
+     * default constructor
```

Comment:
Please write a meaningful comment.  This is worse than nothing.  (Though a good comment is better than nothing, even if the comment is short.)

Reply:
> This is worse than nothing

The original code had nothing, and I would have preferred to keep it, but that causes require javadoc to fail.

> a good comment is better than nothing, even if the comment is short

I don't think that's true, in this case. I'll try to improve the comment, but there really isn't anything to say about this constructor - all it does is call super.